distribution/asf-release-process-guide.md

```diff
@@ -561,7 +561,7 @@ fully endorsed by the ASF.
 
 ### Cancelling a vote
 
-If for any reason during the Druid PPMC or IPMC vote a blocking issue becomes apparent, a vote should be officially cancelled by sending an email with the following subject line: `[CANCEL][VOTE] Release Apache Druid (incubating) 0.15.1 [RC1]` and the reasons for the cancellation in the body.
+If for any reason during the Druid PPMC or IPMC vote a blocking issue becomes apparent, a vote should be officially cancelled by sending an email with the following subject line: `[CANCEL][VOTE] Release Apache Druid 0.15.1 [RC1]` and the reasons for the cancellation in the body.
```

Comment:
The "IPMC" clause here should be removed, since Druid isn't part of the Incubator anymore.

Please also:

- Remove the "IPMC vote" section and any other Incubator-related stuff in other sections. This includes replacing stuff like "Once a release candidate has passed the incubator PMC vote" with "Once a release candidate has passed the PMC vote". This will probably require re-reviewing the entire document.
- Replace references to "Druid PPMC" with "Druid PMC". (PPMCs are for incubating projects.)
- Remove requirement to notify general@incubator.apache.org of new releases.

Reply:
Nevermind, I just noticed you marked this out of scope. Please ignore the above comment.

Reply:
Ah, yeah, I was planning on updating that in a follow-on PR